tests/src/test/scala/actionContainers/Swift4ActionContainerTests.scala

```diff
@@ -91,4 +91,21 @@ class Swift4ActionContainerTests extends SwiftActionContainerTests {
     })
   }
 
+  it should "receive a large (1MB) argument" in {
+    withActionContainer() { c =>
+      val code = """
+                   | func main(args: [String: Any]) -> [String: Any] {
+                   |     return args
+                   | }
+                   |""".stripMargin
+
+      val (initCode, initRes) = c.init(initPayload(code))
+      initCode should be(200)
+
+      val arg = JsObject("arg" -> JsString(("a" * 198144)))
```

Comment:
Hmm, double check that this is actually a MB.

Reply:
Is this 1024 * 1024?

Reply:
humm checking...

Reply:
Looks likes the highest I can go without an margin of error is `1048561` will update test